resources/location.js

```diff
@@ -28,9 +28,9 @@ assign(Location.prototype, omit(base, ['create', 'delete', 'update']));
  * @return {Promise} Promise that resolves with the result
  * @public
  */
-Location.prototype.inventoryLevels = function inventoryLevels(id) {
+Location.prototype.inventoryLevels = function inventoryLevels(id, params) {
```

Comment:
Can you please add JSDoc for the `params` argument?

Reply:
Sure thing, done. 👍